sdk/eventhub/azure-eventhub/azure/eventhub/aio/_client_base_async.py

```diff
@@ -194,22 +189,12 @@ async def _create_auth_async(self) -> authentication.JWTTokenAsync:
         except AttributeError:
             token_type = b"jwt"
         if token_type == b"servicebus.windows.net:sastoken":
-            auth = authentication.JWTTokenAsync(
+            return JWTTokenAuthAsync(
                 self._auth_uri,
                 self._auth_uri,
                 functools.partial(self._credential.get_token, self._auth_uri),
-                token_type=token_type,
-                timeout=self._config.auth_timeout,
-                http_proxy=self._config.http_proxy,
-                transport_type=self._config.transport_type,
-                custom_endpoint_hostname=self._config.custom_endpoint_hostname,
-                port=self._config.connection_port,
-                verify=self._config.connection_verify,
-                refresh_window=300,
```

Comment:
We don't need all these settings?

Reply:
there're advanced feature which are not supported in the python amqp yet.
I have created an issue for it: https://github.com/Azure/azure-sdk-for-python/issues/22550